File: FindtheDifference.cpp
#include<iostream>
#include<vector>

using namespace std;

class Solution {
public:
    char findTheDifference(string s, string t) {
        vector<int> m(26, 0);
        for(char c: s) {
        	m[c - 'a']++;
		}
		for(char c: t) {
			if(m[c - 'a']-- == 0) return c;
		}
		
		return 0;
    }
};

int main() {
	
	string s, t;
	cin>>s>>t;
	
	Solution *solution = new Solution();
	cout<<solution->findTheDifference(s, t);
	
	return 0;
	
}

File: FindtheDifference.py
class Solution:
    def findTheDifference(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: str
        """
        diff = 0
        diff += ord(t[-1])
        for i in range(len(s)):
            diff -= ord(s[i])
            diff += ord(t[i])
        return chr(diff)


if __name__ == '__main__':
    s = input()
    t = input()
    solution = Solution()
    print(solution.findTheDifference(s, t))
